charts/pulsar-operator/README.md

```diff
@@ -46,3 +46,12 @@ To uninstall the `pulsar-operator` chart, execute this command.
 ```
 helm uninstall pulsar-operator -n <k8s-namespace>
 ```
+
+
+## Versioning Convention
+
+`version`: The version of the chart itself, it will change only when there are some changes to the chart itself or `appVersion` bumps a new version.
+
+`appVersion`: The version of the application image that this chart contains, it will change only when operator image bumps a new version.
```

Comment:
```suggestion
`appVersion`: The version of the application image that the chart contains. It will be changed only when the operator image bumps a new version.
```